### ISSUE TITLE
The Manager dont have the Close/Minimise/FullScr as usual. Pls help to check

### ISSUE THREAD

**Issue by johnthanhvu** [downthemall/downthemall-legacy#263]
### Your DownThemAll! and Browser Version and Language and Operating System
Windows 7/ current version as at Oct-7-2017

### Expected Behavior
The manager screen should have Close/Minimise/full screen button

### Actual Behavior
Nothing appear and not able to close/minimal the manager window

### Additional info such as log extracts
*Please enable Diagnostic Logging in the Privacy tab of the DownThemAll! preferences and either copy/paste essential part of the log or attach the entire log. Please note that the log may include sentitive information which you have to remove prior to posting it!*

**Comment by rugehezhishang** [comment]
我too，管理器的标题蓝条is missing，win7 64 ，当时FF 55.0 zh-cn，大概，或者55.1？插件都保持最新。记得当时对系统做的最大动作是卸载了所有的JAVA旧版本并安装了最新的，与以往升级FF时不同的情景是安装新JAVA之前我把注册表里面所有的旧JAVA信息删掉了，包括看懂的和看不懂的，然后现象是查看autoruns里过半数硬件驱动显示文件找不到，但实际系统照常运行。这是第二个现象，同样的是，照常运行，除了需要点击下右键来关闭管理器窗口。不知道是不是与JAVA有关，仅为反馈及求助。
Translated by bing and my muscles：ME too,  the title blue bar of the Manager  is missing , win7 64, FF 55.0 zh-CN, probably, or 55.1? Plugins are keep up to last date .What I Remember the biggest action to the system is to uninstall all old versions of JAVA and install the latest,  different of the past scenarios is  before i install the new JAVA ,i had all the old JAVA registry informations deleted, including understand and not understand, Then is to see a majority,above half of the hardware drivers info in the Autoruns shows file not found, but the actual system running as usual. This is the second, the samething is, running as usual, in addition to need a right click to close the Management window.Don't know if it's JAVA-related, just feedback and requests for assistance.

**Comment by skymariner** [comment]
![adsiz](https://user-images.githubusercontent.com/32845093/31619845-5f7290c2-b29e-11e7-8f72-068482fa2c41.jpg)

it looks like this and does not grow by clicking on the window

**Comment by rugehezhishang** [comment]
so,
![snap 2017-10-19 at 10 08 58](https://user-images.githubusercontent.com/32613236/31751191-b970b62a-b4b6-11e7-8a8d-7d7903837a7a.png)
I do not want to reinstall the system again, or JAVA, or FF, It just looks like a monk not wearing a hat , but works well and nothing wrong else.

**Comment by EleniaB** [comment]
Hello
I have exactly the same problem since the update of FF in September.
In the up right corner the buttons minimise maximise close are functionnals but they are invisible and the zone to click is very lowest; perhaps 10 pixels.
The most difficult to live with this is that is impossible to move the window.
Join the log file hoping that help to solve the problem.
Thank you
[log.txt](https://github.com/downthemall/downthemall/files/1425542/log.txt)

**Comment by rubytube** [comment]
Hi,
same issue for me.
And one comment, I liked the more compact window / buttons in previous versions much, much
better as it didnt take so much vertical place on the screen.
It would be nice if this could at least being made configurable.
If the fix would come and eventually this change, then I will happily donate to the project.

**Comment by kileytoo** [comment]
I'm having the same issue... The three buttons do work, but they are very tiny. The corresponding icons appear only on mouse over. However, my main problem is that I cannot move the window because the title bar appears to be completely missing. I can resize the window but there is no area where I can click and drag to move it. More details: using Firefox 56.0.2 (upgrade to 57 not an option). I tried to downgrade DTA but no previous version compatible with 56 is available. I may consider switching to 55 or lower.

![untitled-1](https://user-images.githubusercontent.com/12149850/33511952-218be400-d726-11e7-9dca-25d35958d377.jpg)

**Comment by rugehezhishang** [comment]
我已经升级到了57，现在使用Aira2及其插件，不如downthemall方便。
bing say：
I've upgraded to 57.Now using AIRA2 and its plugins.It's not convenient like downthemall.

**Comment by NDR777** [comment]
I have the same problem.

**Comment by EleniaB** [comment]
Yes with the firefox 57 down them all is die. And impossible to convert it as WebExtension, the only format that know firefox now. RIP down them all. *sob
